integration-tests/pages/createReport/incidentDetailsPage.js

```diff
@@ -65,6 +65,7 @@ const incidentDetailsPage = () =>
     saveAndReturn: () => {
       cy.get('[data-qa="save-and-return"]').click()
     },
+    clickContinueOrReturn: directionFollowingSave => cy.get(`[data-qa=${directionFollowingSave}]`).click(),
```

Comment:
directionFollowingSave refers to the link user clicked in the incident-details page. There are 2 links on that page  - save and continue & save and return.
If user selects save and continue, their intention is to navigate to the staff-involved page. 
If they select save and return, their intention is to navigate to the task list page. 
The users intentions are saved so that the correct path can be followed even if they are presented with the report-may-already-exist page